editor/constants.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Internal dependencies
  */
-import scssVariables from '!!sass-variables-loader!./assets/stylesheets/_variables.scss';
+import scssVariables from '!!sass-variables-loader!../edit-post/assets/stylesheets/_variables.scss';
```

Comment:
It looks like some SCSS variables should remain locate in `editor/` folder.

Reply:
Yeah, we have this issue for all components, these variables and mixins are not modularized properly. They are used in all the modules "components, "block", "editor", "edit-post" and a lot of them are specific to a module.

Reply:
We should tackle this separately in that case.